elastica_pub_sub:
  ros__parameters:
    queue_size: 10
    print_params: 1
    pub_frequency: 2.0
    